bangc-ops/mlu_op.h

```diff
@@ -1055,6 +1055,32 @@ mluOpSetSparseConvolutionDescriptor(mluOpSparseConvolutionDescriptor_t desc,
                                     const int transpose,
                                     const int inverse);
 
+// Group:: GetIndicePairs
+/*!
+ *  @brief The parameter num_act_out will be obtained from the mluOpSparseConvolutionStruct.
```

Comment:
Obtain num_act_out from mluOpSparseConvolutionStruct. This interface should be called after ::mluOpGetIndicePairs.